src/dataService/utils.ts

```diff
@@ -32,15 +32,7 @@ export function mysqlDateConvert(mysqlDate: Date | string | null): Date | null {
  * @returns the unique values of `input`
  */
 export function uniqueArray<T extends string | number>(input: T[]): T[] {
-  const keys = new Set();
-  const uniqueValues = [];
-  input.forEach((element) => {
-    if (!keys.has(element)) {
-      keys.add(element);
-      uniqueValues.push(element);
-    }
-  });
-  return uniqueValues;
+  return Array.from(new Set(input));
```

Comment:
technically this performs slower but since we're constrained to 30 elements it's fine

Reply:
ya 30 is small amount 
asking for learning: i am seeing both method process list with O(n) and add them to set, and adding to set would be O(1) so they both should take same time right?

Reply:
I think the time complexity is the same, but I looked at some benchmarking tests about the multiple copying (array + set creation) which was slower than starting with an empty array and pushing values to the set. but as I said I think being readable is preferable here since it's not huge data, so I agree with this change!